ML-RHDEVDOCS-3382: Updated the Pipeline builder instances (#57392)
openshift/openshift-docs: cicd/pipelines/working-with-pipelines-using-the-developer-perspective.adoc

In the *Developer* perspective:

> * Use the *Add* -> *Pipeline* -> *Pipeline builder* option to create customized pipelines for your application.

Reviewer: @masauso-lungu How is it appearing in the UI? Please check and do the needful accordingly.

* Use the *Add* -> *From Git* option to create pipelines using operator-installed pipeline templates and resources while creating an application on {product-title}.

> After you create the pipelines for your application, you can view and visually interact with the deployed pipelines in the *Pipelines* view. You can also use the *Topology* view to interact with the pipelines created using the *From Git* option. You need to apply custom labels to a pipeline created using the *Pipeline builder* to see it in the *Topology* view.

Reviewer: ```suggestion
After you create the pipelines for your application, you can view and visually interact with the deployed pipelines in the *Pipelines* view. You can also use the *Topology* view to interact with the pipelines created using the *From Git* option. You must apply custom labels to a pipeline created using *Pipeline builder* to see them in the *Topology* view.
```

Reviewer 2: Is this resolved? If so, please resolve the conversation.
